metecho/api/models.py

```diff
@@ -802,7 +838,8 @@ def finalize_status_completed(self, pr_number, *, originating_user_id):
         self.has_unmerged_commits = False
         self.pr_number = pr_number
         self.pr_is_open = False
-        self.epic.has_unmerged_commits = True
+        if self.epic:
+            self.epic.has_unmerged_commits = True
```

Comment:
We're not doing anything here when the Task is attached to a Project

Reply:
Hm, I think that's correct, since when the Task is merged it goes straight into the Project branch.